pkg/certificates/certificates.go

```diff
@@ -298,29 +298,54 @@ func (authDB *AuthorizationDB) Get(userID string) (Authorizations, error) {
 }
 
 // UserIDs returns a list of all userIDs present in the authorization database.
-func (authDB *AuthorizationDB) UserIDs() ([]string, error) {
-	keys, err := authDB.DB.List([]byte{}, 0)
-	if err != nil {
-		return nil, ErrAuthorizationDB.Wrap(err)
-	}
-	return keys.Strings(), nil
+func (authDB *AuthorizationDB) UserIDs() (userIDs []string, err error) {
+	err = authDB.DB.Iterate(storage.IterateOptions{
+		First:   []byte{},
+		Recurse: true,
+		Prefix:  storage.Key(""),
```

Comment:
You can skip defining `First` and `Prefix` if you don't need them.

Reply:
good to know - thanks!